app/templates/src/test/resources/config/_application.yml

```diff
@@ -11,6 +11,9 @@ async:
     maxPoolSize: 50
     queueCapacity: 10000
 
+liquibase:
+    context: test
```

Comment:
Unless I am mistaken, I add those line in base configuration in the `test/resources`package. When I run my test this file will be use instead of the config files in the `main/resources`. 

When I run my test before to make the PR, the behavior was:
- Run JUnit, context `test`was set in LiquiBase
- Run in dev profile, context `development` was set 
- Run in prod profile, context `production` was set 

If I make a mistake, where I set the `test` context?